zcash_client_backend/src/data_api/testing.rs

```diff
@@ -100,7 +100,6 @@ pub struct TransactionSummary<AccountId> {
     memo_count: usize,
     expired_unmined: bool,
     is_shielding: bool,
-    raw: Vec<u8>,
```

Comment:
The `raw` removal is in the fourth commit (bb4dd304586f694395432a9e18df820dd0e2dc28) instead of the third, but the fix is probably to rewrite the fourth commit's message (and separately fixup or remove the third commit per my previous comment).